frontend/src/components/program/picasso/Picasso.vue

```diff
@@ -192,14 +146,17 @@ export default {
   // emitted events
   emits: [
     'newEntry', // triggered once when a new entry was created via drag & drop (parameters: startTimestamp, endTimestamp)
+    'unlockReminder', // triggered when we think someone is trying to create/move in non-editable mode
   ],
 
   // composition API setup
-  setup(props, { emit, refs }) {
+  setup(props, { emit }) {
     const { editable, scheduleEntries } = toRefs(props)
 
     const isSaving = ref(false)
 
+    const showAvatars = new URLSearchParams(document.location.search).has('showAvatars')
```

Comment:
To evaluate. I'm not sure yet, if we want to display the avatars at all on the picasso.

Besides, I was just wondering if we want to display the time as well. Something like this:
<img width="200" alt="Bildschirmfoto 2022-12-04 um 18 40 55" src="https://user-images.githubusercontent.com/3001985/205506517-c19b648e-91be-41ce-bf4f-977e325e9c04.png">

But if we want to go full textual representation on the picasso, the time would add yet another layer and its maximal form would then be

**2.4 LA Geländegame**
13:00 bis 17:00
[Linux & Smiley]
Wald hinter dem Haus

And this could be a bit much. What do you think?

Reply:
I think the less information we have to show on every schedule entry, the better. Two lines is sometimes already too much for some very short schedule entries. Google calendar displays at most two lines and condenses them together into one line if the event is very short. Some more references of hand-crafted picassos (to be taken with a grain of salt because they are in PDF/Excel form, which does not always translate perfectly into web UI):
[grobprogramm_lager_kindersport_d.pdf](https://github.com/ecamp/ecamp3/files/10149621/grobprogramm_lager_kindersport_d.pdf)
[02 GP HK.pdf](https://github.com/ecamp/ecamp3/files/10149622/02.GP.HK.pdf)
[de-vorlage_grobprogramm_20180404-appendix.xlsx](https://github.com/ecamp/ecamp3/files/10149623/de-vorlage_grobprogramm_20180404-appendix.xlsx)

There is already so much information on the picasso, visually speaking. Instead of also squeezing the time (and location for that matter) onto each schedule entry, I would make the time more clear in the grid if you think it's unclear, or add a tooltip when hovering which could contain often used information like the duration or other things. Another option for post-MVP would be to have a few different "display modes" of the picasso, like the completion percentage mode in ecamp2, or even customizable display of fields on the picasso.

Reply:
For me Google Calendar renders three lines on desktop and only two on mobile.

Desktop:
Title
Time
Location

Mobile:
Title
Location

I like the idea with the modes. I've tried this a long time ago and think this is probably the way. Also I still believe that it is sensible to show more information on the picasso entries  when they have room for it.

Reply:
I have improved my previous version and think it is better now.
<img width="686" alt="Bildschirmfoto 2022-12-30 um 15 50 31" src="https://user-images.githubusercontent.com/3001985/210083052-473ad519-905f-47e4-9fb1-27e29f648e58.png">

In general, I would like view settings. For the picasso, a layout option would make sense. It could have the following options: dense, normal, or wide. _Dense_ would be the same as now, _Normal_ would show at maximum 3 days on mobile and _Wide_ only 2 days. On desktop it would be optimized based on the options. Also you could toggle avatars, locations and maybe other stuff.

Reply:
For now I've decided to display the avatars beginning from an entry width of 200px. The view settings can be implemented in a next PR.